platform/mellanox/get_component_versions/get_component_versions.py

```diff
@@ -20,22 +20,28 @@
 import subprocess
 import re
 
-from fwutil.lib import PlatformDataProvider
+try:
+    from fwutil.lib import PlatformDataProvider
+except Exception:
+    PlatformDataProvider = None
+
 from sonic_py_common.general import check_output_pipe
 from tabulate import tabulate
 
 COMPONENT_VERSIONS_FILE = "/etc/mlnx/component-versions"
 HEADERS = ["COMPONENT", "COMPILATION", "ACTUAL"]
 COMMANDS_FOR_ACTUAL = {
     "MFT": [["dpkg", "-l"], ["grep", "mft "], "mft *([0-9.-]*)"],
-    "HW-MGMT": [["dpkg", "-l"], ["grep", "hw"], ".*1\\.mlnx\\.([0-9.]*)"],
+    "HW_MANAGEMENT": [["dpkg", "-l"], ["grep", "hw"], ".*1\\.mlnx\\.([0-9.]*)"],
     "SDK": [["docker", "exec", "-it", "syncd", "bash", "-c", 'dpkg -l | grep sdk'], ".*1\\.mlnx\\.([0-9.]*)"],
     "SAI": [["docker", "exec", "-it", "syncd", "bash", "-c", 'dpkg -l | grep mlnx-sai'], ".*1\\.mlnx\\.([A-Za-z0-9.]*)"],
     "FW": [["mlxfwmanager", "--query"], "FW * [0-9]{2}\\.([0-9.]*)"],
-    "Kernel": [["uname", "-r"], "([0-9][0-9.-]*)-.*"]
+    "KERNEL": [["uname", "-r"], "([0-9][0-9.-]*)-.*"],
+    "SIMX": [["lspci", "-vv"], ["grep", "SimX"], "([0-9]+\\.[0-9]+-[0-9]+)"]
```

Comment:
@skr31 SImX is not a component. This looks like a hack...

Reply:
Fixed